table/server/master/src/main/java/alluxio/master/table/DbConfig.java

```diff
@@ -55,153 +97,537 @@ public DbConfig(@JsonProperty("bypass") @Nullable BypassEntry bypassEntry) {
    * @return an empty config instance
    */
   public static DbConfig empty() {
-    return new DbConfig(new BypassEntry(Collections.emptySet()));
+    return new DbConfig(null, null);
   }
 
   /**
-   * @return the {@link BypassEntry} from config file
+   * @return the {@link TablesEntry} for bypassed tables from config file
    */
-  public BypassEntry getBypassEntry() {
+  public TablesEntry<TableEntry> getBypassEntry() {
     return mBypassEntry;
   }
 
   /**
-   * @return the {@link UdbBypassSpec} object
+   * @return the {@link TablesEntry} for ignored tables from config file
    */
-  public UdbBypassSpec getUdbBypassSpec() {
-    return mBypassEntry.toUdbBypassSpec();
+  public TablesEntry<NamePatternEntry> getIgnoreEntry() {
+    return mIgnoreEntry;
+  }
+
+  /**
+   * @return the {@link UdbMountSpec} object
+   */
+  public UdbMountSpec getUdbMountSpec() {
+    UdbMountSpec.Builder builder = new UdbMountSpec.Builder();
+    for (TableEntry entry : mBypassEntry.getList().getIncludedEntries()) {
+      if (entry.isPattern()) {
+        builder.bypass().include().addPattern(entry.getPattern());
+        continue;
+      }
+      if (entry.getPartitions().isEmpty()) {
+        builder.bypass().include().addName(entry.getName());
+        continue;
+      }
+      IncludeExcludeList<NamePatternEntry> partitions = entry.getPartitions();
+      UdbMountSpec.SimpleWrapperBuilder partitionBuilder = new UdbMountSpec.SimpleWrapperBuilder();
+      for (NamePatternEntry partition : partitions.getIncludedEntries()) {
+        if (partition.isPattern()) {
+          partitionBuilder.include().addPattern(partition.getPattern());
+        } else {
+          partitionBuilder.include().addName(partition.getName());
+        }
+      }
+      for (NamePatternEntry partition : partitions.getExcludedEntries()) {
+        if (partition.isPattern()) {
+          partitionBuilder.exclude().addPattern(partition.getPattern());
+        } else {
+          partitionBuilder.exclude().addName(partition.getName());
+        }
+      }
+      builder.bypass().include().addPartition(entry.getTable(), partitionBuilder);
+    }
+
+    for (NamePatternEntry entry : mBypassEntry.getList().getExcludedEntries()) {
+      if (entry.isPattern()) {
+        builder.bypass().exclude().addPattern(entry.getPattern());
+      } else {
+        builder.bypass().exclude().addName(entry.getName());
+      }
+    }
```

Comment:
Exclusion list on the table level does not support partition specification; to exclude a partition, include the table, and exclude that partition from its partition specfifcation, e.g. `{"include": [{"table": "table1", "partition": {"exclude": ["part1"]}}]}`.